packages/exec/__tests__/exec.test.ts

```diff
@@ -153,6 +153,23 @@ describe('@actions/exec', () => {
     expect(output.trim()).toBe(`hello`)
   })
 
+  it('Runs exec successfully with shell wildcard substitution', async () => {
+    let tool: string
+    let args: string[]
+    let opts: im.ExecOptions
+    if (IS_WINDOWS) {
+      tool = 'cmd'
+      args = ['/c', 'echo', 'hello']
+      opts = {shell: process.env.ComSpec}
```

Comment:
Suggest also adding a test for running bash shell within Windows. As bash is now supported by windows runners, it provides a universal target for actions wanting to execute shell commands.

Reply:
ok